configs/variables

```diff
@@ -13,9 +13,9 @@ FEDORA_HOME="/usr/local/fedora"
 FFMPEG_VERSION=1.1.4
 
 FITS_HOME="/usr/local/fits"
-FITS_VERSION=0.10.1
+FITS_VERSION=1.1.1
 
-SOLR_HOME="/usr/local/solr"
-SOLR_VERSION=4.2.0
+SOLR_HOME="/usr/local/fedora/solr"
```

Comment:
Hey People, why did you change the Solr home directory? Being it inside fedora is a bit confusing. Are there any strict devops requirements for this?